cli/utils/api.go

```diff
@@ -138,8 +138,16 @@ func (resourceClient ResourceClient) Delete(ctx context.Context, ID string) erro
 		return fmt.Errorf("could not delete resource: %w", err)
 	}
 
-	_, err = resourceClient.Client.Do(request)
-	return err
+	response, err := resourceClient.Client.Do(request)
+	if err != nil {
+		return err
+	}
+
+	if response.StatusCode == http.StatusNotFound {
+		return fmt.Errorf(`no resource with ID "%s" was found`, ID)
```

Comment:
Can we use the same message on the `get` command [here](https://github.com/kubeshop/tracetest/blob/main/cli/cmd/get_cmd.go#L46)?

```suggestion
		return fmt.Errorf(`Resource %s with ID %s not found" was found`, resourceClient.ResourceType, ID)
```